sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -28,7 +28,7 @@ multiapi: true
 batch:
```

Comment:
you can add `python3-only: true` under the basic config so autorest doesn't generate models for python 2.